### ISSUE TITLE
Ability to configure fiat currency

### ISSUE THREAD

**Issue by chkpnt** [C-Otto/BitBook#11]
Currently, `BitBooks` is only able to show the fiat values in EUR.

**Comment by C-Otto** [comment]
Yes, I guess USD would be a good next step. Do you know any good API candidate?

**Comment by chkpnt** [comment]
I've seen you are requesting Kraken, so the easist way would be to use Kraken for BTCUSD, too.

Don't know any free (registration-less) api for historical data. As a user, I usally ask wolframalpha for that purpose, but don't know anything about their API...

<img width="1011" alt="grafik" src="https://user-images.githubusercontent.com/1956979/114105653-6bd87e80-98cd-11eb-8341-23089042e0b1.png">
